.github/workflows/make_release.yml

```diff
@@ -6,6 +6,10 @@ on:
       tag:
         description: Manual Tag
         required: true
+      type:
```

Comment:
Where is the script change to add it to e2e-test workflow?

Reply:
What do you mean?

Reply:
How to get this triggered from e2e-test.yaml - something similar to non fips release? the tag would be different and we need to set that up and trigger - right?